### PR TITLE
STORM-3791 update metric documentation

### PR DESCRIPTION
**Pull request by agresch**

### CODE REVIEW

File: docs/Metrics.md [hunk 1]
```diff
@@ -213,37 +213,33 @@ This metric records how many errors were reported by a spout/bolt. It is the tot
...
 
 `arrival_rate_secs` is an estimation of the number of tuples that are inserted into the queue in one second, although it is actually the dequeue rate.
 The `sojourn_time_ms` is calculated from the arrival rate and is an estimate of how many milliseconds each tuple sits in the queue before it is processed.
-Prior to STORM-2621 (v1.1.1, v1.2.0, and v2.0.0) these were the rate of entries, not of tuples.
 
-A disruptor queue has a set maximum number of entries.  If the regular queue fills up an overflow queue takes over.  The number of tuple batches stored in this overflow section are represented by the `overflow` metric.  Storm also does some micro batching of tuples for performance/efficiency reasons so you may see the overflow with a very small number in it even if the queue is not full.
+The queue has a set maximum number of entries.  If the regular queue fills up an overflow queue takes over.  The number of tuple batches stored in this overflow section are represented by the `overflow` metric.  Storm also does some micro batching of tuples for performance/efficiency reasons so you may see the overflow with a very small number in it even if the queue is not full.
```
Comment (Ethanlm) [inline docs/Metrics.md:238]: I am not very sure about this sentence:
```
Storm also does some micro batching of tuples for performance/efficiency reasons so you may see the overflow with a very small number in it even if the queue is not full.
```
Is this still valid?

**Review by Ethanlm — COMMENTED** [review]
Overall looks good. Thanks for updating the docs

File: docs/Metrics.md (hunk 1 above)
Comment (agresch) [reply docs/Metrics.md]: Not aware of anything.  Removed it.

File: docs/Metrics.md [hunk 2]
```diff
@@ -235,7 +235,7 @@ These queues report the following metrics:
 `arrival_rate_secs` is an estimation of the number of tuples that are inserted into the queue in one second, although it is actually the dequeue rate.
 The `sojourn_time_ms` is calculated from the arrival rate and is an estimate of how many milliseconds each tuple sits in the queue before it is processed.
 
-The queue has a set maximum number of entries.  If the regular queue fills up an overflow queue takes over.  The number of tuple batches stored in this overflow section are represented by the `overflow` metric.  Storm also does some micro batching of tuples for performance/efficiency reasons so you may see the overflow with a very small number in it even if the queue is not full.
+The queue has a set maximum number of entries.  If the regular queue fills up an overflow queue takes over.  The number of tuple batches stored in this overflow section are represented by the `overflow` metric.
```
Comment (Ethanlm) [inline docs/Metrics.md:238]: Remove `batches` in `tuple batches` too? If I understand correctly, tuples are inserted to the queue one by one.   

Maybe add the following: 

Note that an overflow queue is only used for executors to receive tuples from remote workers. It doesn't apply to intra-worker tuple transfer.

Comment (agresch) [reply docs/Metrics.md]: updated

**Review by Ethanlm — APPROVED** [review]

**Pull request merged**